ALUM/ALUM/ViewModels/QuestionViewModel.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  QuestionViewModel.swift
+//  ALUM
+//
+//  Created by Neelam Gurnani on 3/9/23.
+//
+
+import Foundation
+
+final class QuestionViewModel: ObservableObject {
+    @Published var questionList: [Question] = []
+    @Published var currentIndex: Int = 0
+    @Published var lastQuestion: Bool = true
+
+    func loadTestData() -> [Question] {
+        var questions: [Question] = []
+
+        var question1 = Question(question: "Testing Question 1", type: "text", id: "1", answerBullet: [],
+                                 answerParagraph: "Testing Answer 1")
+        var question2 = Question(question: "Testing Question 2", type: "text", id: "2", answerBullet: [],
+                                 answerParagraph: "Testing Answer 2")
+        var question3 = Question(question: "Testing Question 3", type: "text", id: "3", answerBullet: [],
+                                 answerParagraph: "Testing Answer 3")
+
+        questions.append(question1)
+        questions.append(question2)
+        questions.append(question3)
+
+        return questions
+    }
+
+    func fetchQuestions() async throws -> [Question] {
+        let url = URL(string: "http://localhost:3000/notes/")!
```

Comment:
is this to get the actual questions themselves? i don't think we're planning to host the questions in the backend, i think they're just gonna be hard coded in.

unless this is to get the answers? if so we need an id after the notes as well.

Reply:
We are planning to host questions on the backend haha. We can discuss this in our meeting but this is the response the GET notes route should return
```
GET Notes
URL - /notes/<id>
Response Format
[
    {
        question: "Question?",
        type: "text",
        id: "the hashed ID 1"
    }, 
    {
        question: "Question?",
        type: "bullet",
        id: "the hashed ID 2"
    }
]
```

Reply:
Can all backend interactions be consolidated into one API service class? #69 adds an `APIServices` file, perhaps use something like that